.github/workflows/tests.yml

```diff
@@ -7,18 +7,16 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - python-version: '3.6'
```

Comment:
@sneridagh wondering if we can remove 3.6 here. If Plone 5.2 officially supports it, we should continue to test it here. Or am I missing something?

Reply:
"Supports Python 2.7, 3.6, 3.7, 3.8. Note that both Python 2.7 and 3.6 have reached end of life, and will get harder to test and support."

https://plone.org/download/release-schedule#:~:text=Current%20release%3A%205.2.,%2C%203.6%2C%203.7%2C%203.8.

Reply:
In that case ok, wondering if we have to review that policy given the current expectative and the fact that Python is releasing a major every year.

Reply:
Plone 5.2 has dropped 3.6 support. See my announcement earlier this week: https://community.plone.org/t/plone-5-2-drops-python-3-6-support/15706
I have updated the text on the release schedule now.